docker/bin/certify.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+DOCKER_BIN="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+export PASS="abcdefgh"
+source ${DOCKER_BIN}/../.env
+
+if [ -f ${TLS}/ca-cert ]; then
```

Comment:
use new style if brackets to avoid having to escape stuff:
`if [[ -f .. ]] ; then`

Reply:
Aren't their portability concerns with `[[ ]]`? 

Seems more prudent to stick with the tried-and-true test command

Reply:
double brackets were added to bash 2.02 back in 1998, I think we can safely assume they're supported.

It is good practice to use them since it removes a lot of corner cases with regards to quoting and escaping.